include/magic_enum_containers.hpp

```diff
@@ -591,7 +591,7 @@ class bitset {
   }
 
   template <typename Cmp = std::equal_to<>>
-  constexpr explicit bitset(string_view sv, Cmp&& cmp = {}, char sep = '|') {
+  constexpr explicit bitset(string_view sv, Cmp&& cmp = {}, char_type sep = '|') {
```

Comment:
static_cast<char_type>('|')
Check this character too at static_assert check at magic_enum.hpp line 141/142